packages/salesforcedx-vscode-lwc/test/vscode-integration/intellisense.test.ts

```diff
@@ -8,217 +8,264 @@
 import { assert, expect } from 'chai';
 import * as path from 'path';
 import * as vscode from 'vscode';
+import {
+  commands,
+  CompletionItem,
+  CompletionItemKind,
+  CompletionList,
+  extensions,
+  Extension,
+  Position,
+  Selection,
+  TextDocument,
+  TextEditor,
+  Uri,
+  window,
+  workspace
+} from 'vscode';
 
-describe('LWC Intellisense Test Suite', function() {
-  let lwcDir: string;
-  let lwcExtension: vscode.Extension<any>;
-  before(async() => {
-    lwcDir = path.join(
-      vscode.workspace.workspaceFolders![0].uri.fsPath,
-      'force-app',
-      'main',
-      'default',
-      'lwc'
-    );
+describe('LWC Intellisense Integration Tests', () => {
 
-    lwcExtension = vscode.extensions.getExtension(
-      'salesforce.salesforcedx-vscode-lwc'
-    ) as vscode.Extension<any>;
-    await lwcExtension.activate();
-  });
+  describe('LWC JS Intellisense Test Suite', function() {
+    // tslint:disable-next-line:no-invalid-this
+    this.timeout(20000);
+    let lwcDir: string;
+    let lwcExtension: Extension<any>;
+    let docUri: Uri;
+    let doc: TextDocument;
+    let editor: TextEditor;
+    let text: string;
+    let startPosition: Position;
+    let endPosition: Position;
+    let selection: Selection;
 
-  afterEach(async () => {
-    try {
-      await vscode.commands.executeCommand(
-        'workbench.action.closeActiveEditor'
+    beforeEach(async() => {
+      lwcDir = path.join(
+        vscode.workspace.workspaceFolders![0].uri.fsPath,
+        'force-app',
+        'main',
+        'default',
+        'lwc'
       );
-    } catch (e) {
-      throw e;
-    }
-  });
+      lwcExtension = extensions.getExtension(
```

Comment:
Should this and the next line go in a before function for the top level describe?   It should only have to verify it's activated once.